eng/Versions.props

```
@@ -1,8 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <VersionPrefix>0.1.0</VersionPrefix>
-    <PreReleaseVersionLabel>alpha1</PreReleaseVersionLabel>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <!-- 
+      For sorting purposes, it is very important there be a '.' between the word 'preview' and the preview number.
+      That ensures that if we get up to preview 10, it doesn't sort as between preview 1 and preview 2!
```

```suggestion
      That ensures that if we get up to preview 10, it doesn't sort as between preview 1 and preview 2!
```

I aspire to 10 previews.